themes/bootstrap5/templates/_ui/components/show-sidebar-button.phtml

```diff
@@ -1,9 +1,9 @@
 <?php if (null === ($side = $this->config()->offcanvasSide())): ?>
-    <a class="search-filter-toggle" href="#<?=$this->escapeHtmlAttr($this->targetId)?>" title="<?=$this->transEscAttr($this->title ?? 'sidebar_expand') ?>">
+    <a class="search-filter-toggle <?=$this->escapeHtmlAttr($this->_themeClass);?>" href="#<?=$this->escapeHtmlAttr($this->targetId)?>" title="<?=$this->transEscAttr($this->title ?? 'sidebar_expand') ?>">
```

Comment:
Any particular reason you converted the other templates to use the `htmlAttributes` helper but left this alone? It's fine as-is, but conversion might improve readability here as well.